README.md

```diff
@@ -116,6 +116,69 @@ class Site < Granite::ORM::Base
 end
 ```
 
+### Bulk Insertions
+
+#### Import
+
+Each model has an `import` class level method to import an array of models in one bulk insert statement.
+```Crystal
+models [
+  Model.new(id: 1, name: "Fred", age: 14),
+  Model.new(id: 2, name: "Joe", age: 25),
+  Model.new(id: 3, name: "John", age: 30),
+]
+
+Model.import(models)
+```
+
+#### on_duplicate_key_update
+
+The `import` method has an optional `on_duplicate_key_update` param allows you to specify fields (as an array of strings) to be updated should a primary constraint is violated.
+```Crystal
+models [
+  Model.new(id: 1, name: "Fred", age: 14),
+  Model.new(id: 2, name: "Joe", age: 25),
+  Model.new(id: 3, name: "John", age: 30),
+]
+
+Model.import(models)
+
+Model.find!(1).name # => Fred
+
+models [
+  Model.new(id: 1, name: "George", age: 14),
+]
+
+Model.import(models, on_duplicate_key_update: %w(name))
```

Comment:
This `on_duplicate_key_update` parameter feels kind of strange, I think for two reasons. One, it's sort of a double purpose: "on duplicate update?" and "update using these fields". Second, it's really close to the the sql that it's using behind the scenes, which sort of breaks the veil that the ORM is using to obscure the dev from sql.

What if we could write something like:

```crystal
Model.import models, update_on_duplicate: true, with_fields: %(name, ...)
```

Both fields could be required with each other, so you'd never get just one of them and have a valid method call, and in Crystal we have the benefit of being able to enforce that with the compiler. Does this make it more readable, especially to someone unfamiliar with the library?

Reply:
I could go with that, also adds consistency to the options, as both `update_on_duplicate` and `ignore_on_duplicate` would be booleans.

Reply:
Bah, it seems like I already commented on this below...sorry